[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/pre-configuration-database-optimizations1.md

## Set NTFS File Allocation Unit

>  SQL Server stores its data in **Extents**, which are groups of eight 8K pages. Therefore, to optimize disk performance, set the NTFS Allocation Unit size to 64KB as described in the “Disk Configuration Best Practices” section of the SQL Server best practices article [Predeployment I/O Best Practices](https://go.microsoft.com/fwlink/?LinkId=140818). For more information about SQL Server pages and extents see the SQL Server Books Online topic [Understanding Pages and Extents](/previous-versions/sql/sql-server-2008-r2/ms190969(v=sql.105)) ( HYPERLINK "<https://go.microsoft.com/fwlink/?LinkId=148939>" <https://go.microsoft.com/fwlink/?LinkId=148939>).

Reviewer: ```suggestion
 SQL Server stores its data in **Extents**, which are groups of eight 8K pages. Therefore, to optimize disk performance, set the NTFS Allocation Unit size to 64KB as described in the SQL Server guide for ["Disk Configuration Best Practices - Predeployment I/O Best Practices](/previous-versions/sql/sql-server-2005/administrator/cc966412(v=technet.10)#disk-configuration-best-practices--common-pitfalls). For more information about SQL Server pages and extents, see [Understanding Pages and Extents](/previous-versions/sql/sql-server-2008-r2/ms190969(v=sql.105)).
```

## Database planning considerations

>  We recommend that you host your [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] databases on fast storage (for example, fast SAN disks or fast SCSI disks). We recommend RAID 10 (1+0) instead of RAID 5 because raid 5 is slower at writing. Newer SAN disks have very large memory caches, so in these cases, the RAID selection is not as important. To increase performance, databases and their log files can reside on different physical disks.

Reviewer: ```suggestion
Microsoft recommends that you host your [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] databases on fast storage (for example, fast SAN disks or fast SCSI disks). Microsoft recommends RAID 10 (1+0) instead of RAID 5 because RAID 5 is slower at writing. Newer SAN disks have very large memory caches, so in these cases, the RAID selection is not as important. To increase performance, databases and their log files can reside on different physical disks.
```

>  Implementing Trace Flag –T1118 helps reduce contention across the SQL Server instances by removing almost all single page allocations. For more information, see Microsoft Knowledge Base article 328551 "PRB: Concurrency enhancements for the tempdb database" at [https://go.microsoft.com/fwlink/?LinkID=103713](https://go.microsoft.com/fwlink/?LinkID=103713).

Reviewer: ```suggestion
If you implement Trace Flag - T1118 helps reduce contention across the SQL Server instances by removing almost all single page allocations. For more information, see [Microsoft Knowledge Base 328551 - Concurrency enhancements for the tempdb database](https://support.microsoft.com/topic/kb328551-concurrency-enhancements-for-the-tempdb-database-ca601d44-4b95-70c7-2afe-df0bf7ba716c).
```
